config-service-api/src/main/proto/org/hypertrace/config/service/v1/config_service.proto

```diff
@@ -161,3 +164,39 @@ message UpsertAllConfigsResponse {
     optional google.protobuf.Value prev_config = 5;
   }
 }
+
+message Filter {
+  oneof type {
+    RelationalFilter relational_filter = 1;
+    LogicalFilter logical_filter = 2;
+  }
+}
+
+message RelationalFilter {
+  string lhs = 1;
```

Comment:
in general, spell out words. the lhs here is the config_json_path right?
(and operator, value for op, rhs).